e2e/app/authenticated-page.ts

```diff
@@ -179,7 +179,7 @@ export default abstract class AuthenticatedPage extends BasePage {
 
   private async isSideNavDropdownOpen(): Promise<boolean> {
     try {
-      await findIcon(this.page, 'Home', 'home', {visible: true, timeout: 1000});
+      await findIcon(this.page, {text: 'Home'}, 'home', {visible: true, timeout: 1000});
```

Comment:
can you make a utility method that's an async boolean-valued method to wait look for and wait for an icon (or other element)? It seems like this would be a common itch to scratch.